bip-nsequence-anti-fee-snipe.mediawiki

```diff
@@ -0,0 +1,112 @@
+<pre>
+  BIP: TBD
+  Layer: Applications
+  Title: Anti-fee-sniping protection with nSequence in taproot transactions to improve privacy for off-chain protocols
+  Author: Chris Belcher <belcher at riseup dot net>
+  Status: Draft
+  Type: Standards Track
+  Created: 2021-06-10
+  License: PD
```

Comment:
I'd suggest to use less ambiguous CC-0 instead, see https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki#not-acceptable-licenses